### DISCUSSION TITLE
A tool within CVAT to convert polyline to polygon (with a fixed width)

### DISCUSSION THREAD

**Discussion by binaryannamolly** [cvat-ai/cvat#4490] [Ideas]
Some features are suited for polyline annotations, like road marks. Not to mention that polyline annotations should be faster to create than e.g. polygon annotations. With that in mind I would have liked to see a feature where one could convert polyline annotations into polygon annotations - with a fixed width (set by user) between polygon border, in number of pixels.

**Comment by bsekachev** [comment]
Hi, that is an interesting feature, please submit an issue.
